oar/core/exceptions.py

```diff
@@ -6,17 +6,25 @@ class WorksheetException(BaseException):
     """Exception class to raise error in WorksheetManager"""
 
 
+class WorksheetExistsException(WorksheetException):
+    """Exception class to raise workhseet already exists error"""
+
+
```

Comment:
Are the new lines intended? There are two now between the classes.

Reply:
it is formatted by plugin `"editor.defaultFormatter": "ms-python.black-formatter"`

Reply:
```
"[python]": {
    "diffEditor.ignoreTrimWhitespace": false,
    "gitlens.codeLens.symbolScopes": ["!Module"],
    "editor.defaultFormatter": "ms-python.black-formatter",
    "editor.formatOnSave": true,
    "editor.autoClosingBrackets": "always"
  }
```
this is my vscode config for python code formatter

Reply:
Ok, thanks 🙂. I was not sure if it is a auto-formatter or a typo.